src/sass/_theme_rst.sass

```diff
@@ -200,14 +200,14 @@
   // This is the #href that shows up on hover. Sphinx's is terrible so I hack it away.
   h1, h2, h3, h4, h5, h6, dl dt, p.caption, table > caption, .code-block-caption
     .headerlink
-      visibility: hidden
-      font-size: 14px
+      opacity: 0
```

Comment:
This is the key. `display: none` and `visibility: hidden` remove the element from the tab order, whereas making the element transparent doesn't